packages/notifications-flyout/src/presenters/IndicatorPresenter.scss

```diff
@@ -24,9 +25,3 @@
     display: none;
   }
 }
-
```

Comment:
just wanted to make sure this removal was intentional since it looks like the main goal was to add `display: flex;`